src/app/staking/components/delegates/DelegateCardList.tsx

```diff
@@ -0,0 +1,119 @@
+"use client";
+
+import { useEffect, useRef, useState } from "react";
```

Comment:
these were different than the current delegate cards we have, right? I recall discussing this with you but want to be sure thats why we aren't just using the existing ones